sw/device/silicon_creator/mask_rom/mask_rom_epmp.c

```diff
@@ -102,6 +102,29 @@ void mask_rom_epmp_unlock_rom_ext_rx(epmp_state_t *state, epmp_region_t image) {
   CSR_SET_BITS(CSR_REG_PMPCFG1, kEpmpModeTor | kEpmpPermLockedReadExecute);
 }
 
+void mask_rom_epmp_config_rom_ext_r(epmp_state_t *state, epmp_region_t image) {
```

Comment:
minor nit: Would 
```suggestion
void mask_rom_epmp_config_rom_ext_r(epmp_state_t *state, epmp_region_t region) {
```
or something similar be clearer since the area we unlock is greater than the image.

Reply:
done